src/introspective.jl

```diff
@@ -1,4 +1,9 @@
-CACHE = (0, [])
+mutable struct GlobalCache
+  time::Float64
+  cache::Vector{DocObj}
+end
+const CACHE = GlobalCache(time(), DocObj[])
```

Comment:
Ah I figured out. So `GlobalCache.time` is bound to the time when this package is precompiled, but if test happens in a row in quick, the cache would work incorrectly.

Fix soon.